nodebuilder/core/flags.go

```diff
@@ -8,9 +8,12 @@ import (
 )
 
 var (
-	coreFlag     = "core.ip"
-	coreRPCFlag  = "core.rpc.port"
-	coreGRPCFlag = "core.grpc.port"
+	coreFlag           = "core.ip"
+	coreRPCFlag        = "core.rpc.port"
```

Comment:
if you rebase, you won't need this anymore and also won't need to specify `grpc` in the name of the flag

so `core.grpc.tls.path` --> `core.tls.path` and same for xtoken